### PR TITLE
Import Info Serializer #384

### PR DESCRIPTION
**Pull request by tishiljk3**
## Description

Serializer for ImportInfo
## Issue ticket number and link
closes [384](https://github.com/USEPA/haztrak/issues/384)

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by dpgraham4401 — CHANGES_REQUESTED** [review]
Every looks great, one minor change to the capitalization of the port of entry field.

**Comment by dpgraham4401** [comment]
@tishiljk3 @rahulrameshan just a tip (you can take it or leave it)

We prefer if our contributors create a git branch, make your edits in that new branch, push that branch to GitHub, and create the PR from that branch instead of the 'main'. 

When creating the PR, we also prefer that the "Allow edits from maintainers" be checked (set to yes).

 With this setup, if there's something small that needs to change before the PR can be merged, we can still merge the PR if the contributors don't have time to make the changes themselves.

I will update our contributors guide to include this info. 

Cheers!

File: server/apps/trak/serializers/manifest_ser.py
Comment (tishiljk3) [reply server/apps/trak/serializers/manifest_ser.py:310]: @dpgraham4401 will make the change, thank you

**Review by dpgraham4401 — APPROVED** [review]
Awesome, thank you @tishiljk3!

**Pull request merged**
